assets/js/radiohelpers.js

```diff
@@ -34,6 +34,9 @@ function frequencyToBand(frequency) {
 	else if(result >= 50000000 && result <= 56000000) {
 		return '6m';
 	}
+	else if(result >= 70000000 && result <= 70500000) {
```

Comment:
I just checked the DARC Country list for the possible range of the 4m band. Seems like we have to make this range a bit wider. For reference you can check Bosnia & Herzogovina

```suggestion
	else if(result >= 68000000 && result <= 75000000) {
```